src/System.Text.Encoding.Extensions/System.Text.Encoding.Extensions.sln

```diff
@@ -3,7 +3,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
 VisualStudioVersion = 14.0.22609.0
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Encoding.Extensions.Tests", "tests\4.0.10.0\Encoding\Encoding.Extensions.Tests.csproj", "{9574CEEC-5554-411B-B44C-6CA9EC1CEB08}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Encoding.Extensions.Tests", "tests\System.Text.Encoding.Extensions.Tests.csproj", "{9574CEEC-5554-411B-B44C-6CA9EC1CEB08}"
```

Comment:
The name of the project should change as well, right? i.e. "System.Text.Encoding.Extensions.Tests" instead of "Encoding.Extensions.Tests"?